BM/app/src/main/java/com/example/bm/userInterfaceJava/mainActivity.java

```diff
@@ -39,19 +38,16 @@ protected void onCreate(Bundle savedInstanceState) {
 
         Email = (EditText)findViewById(R.id.uname);
```

Comment:
The variable uname can be renamed as email for better code.

Reply:
Thank you :)